cloudbuild.yaml

```
@@ -39,6 +39,8 @@ steps:
           npm run build-prod-enterprise;
         elif [ "${_BUILD_ENV}" = "prod-in" ]; then 
           npm run build-prod-in;
+        elif [ "${_BUILD_ENV}" = "prod_agenticfirst" ]; then 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)